docs/_static/css/sphinx_prompt_css.css

```diff
@@ -0,0 +1,13 @@
+/* CSS for sphinx-prompt */
+
+.highlight {
```

Comment:
Just in case, is it possible to _more specific_ to catch only `sphinx-prompt` generated tags here?

We do have a similar class called `highlighted` that it's used on the search and this could be confusing.

(same for the other `.prompt1` class added here)

Reply:
Yes, we can make it more specific.
I have pushed the changes.